recipes/wcurl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "wcurl" %}
+{% set version = "2024.12.08" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/curl/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9c0615b2c5d6b21da79ff559e75452197330d18449085a18e05f4b623b144a94
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - curl >=7.83.0
+    - __unix   # [unix]
+    - __win    # [win]
+    - m2-bash  # [win]
```

Comment:
You are missing step 2 from https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies.

Reply:
Done! Thank you